app/controllers/concerns/unconfirmed_user_concern.rb

```diff
@@ -25,12 +25,12 @@ def track_user_already_confirmed_event
       errors: { email: [t('errors.messages.already_confirmed')] },
       user_id: @user.uuid,
     }
-    analytics.track_event(Analytics::USER_REGISTRATION_EMAIL_CONFIRMATION, hash)
+    analytics.user_registration_email_confirmation(**hash)
```

Comment:
let's inline `hash` variable, no need for it:
```suggestion
    analytics.user_registration_email_confirmation(
      success: false,
      errors: { email: [t('errors.messages.already_confirmed')] },
      user_id: @user.uuid,
    )
```